__tests__/app.test.js

```diff
@@ -14,20 +14,30 @@ afterAll(() => {
 })
 
 describe("Error codes", () => {
-    it("returns a 404 status if summoned with an incorrect api url", () => {
+    it("GET /incorrect-url returns a 404 status", () => {
         return request(app)
             .get('/apx')
             .expect(404)
             .then((response) => {
                 expect(response.res.statusMessage).toBe('Not Found')
             })
     })
-    it("returns a 404 status if summoned with an incorrect topics url", () => {
+    it("GET /api/articles/:article_id returns a 404 status if no articles can be found with ID", () => {
         return request(app)
-            .get('/api/not-topics')
+            .get('/api/articles/99999999')
             .expect(404)
             .then((response) => {
-                expect(response.res.statusMessage).toBe('Not Found')
+                expect(JSON.parse(response.text)).toEqual({ msg: "No articles found"})
+                
+            })
+    })
+    it("GET /api/articles/:article_id returns a 404 status if no comments can be found", () => {
+        return request(app)
+            .get('/api/articles/2/comments')
```

Comment:
we also want to test what error comes back if we are trying to access the comments for an invalid article id eg /api/articles/banana/comments